webui/react/src/pages/ModelRegistry.tsx

```diff
@@ -338,6 +338,9 @@ const ModelRegistry: React.FC<Props> = ({ workspace }: Props) => {
   const workspaceRenderer = useCallback(
     (record: ModelItem): React.ReactNode => {
       const workspace = workspaces.find((u) => u.id === record.workspaceId);
+      if (!workspace) {
+        return <DynamicIcon name="-" size={24} />;
```

Comment:
currently, we can make a workspace named `-`. How do we distinguish them?

Reply:
this could be empty, or text in place of a workspace; it's not in a Link element now but it could be inside a disabled Link